domains/measurements/Measurements.API/Tests/AggregationTests.cs

```diff
@@ -20,16 +20,11 @@ public sealed class AggregationTests
     [InlineData(Aggregation.Year, new long[] { 1609538400 }, new long[] { 1609552799 }, new long[] { 3930 })]
     public void Measurements_CalculateAggregation_ForConsumption(Aggregation aggregation, long[] expectedDateFrom, long[] expectedDateTo, long[] expectedValues)
     {
-        // Arrange
-        var dateFrom = new DateTimeOffset(2021, 1, 1, 22, 0, 0, TimeSpan.Zero);
-        var dateTo = new DateTimeOffset(2021, 1, 2, 1, 59, 59, TimeSpan.Zero);
```

Comment:
Why delete these?

Reply:
The tests don't even use the variables. They declare them, but aren't being used whatsoever.